1_fetch/src/get_sb_data.R

```diff
@@ -206,18 +72,95 @@ download_children <-function(sites, sb_table_reduced, dldir, workdir, outdir, ou
   
   itemfails <- as.data.frame(itemfails)
   
-  filepath1 <- file.path(outdir, out_file_name)
+  filepath1 <- paste0(outdir, "/", out_file_name, item, ".csv")
   write_csv(data_at_sites, filepath1)
-  filepath2 <- file.path(outdir, paste0("FAILS_", out_file_name))
+  filepath2 <- paste0(outdir, "/", out_file_name, item, "_FAILS.csv")
   write_csv(itemfails, filepath2)
   return_df = c(filepath1, filepath2)
   return(return_df)
 }
 
-calc_avg_dams <- function(sb_data) {
-  sb_data <- read_csv(sb_data, show_col_types = FALSE) %>%
-    suppressMessages()
-  dams <- sb_data %>%
+get_sb_data_log_g2 <- function(sb_var_ids, file_out) {
```

Comment:
Is this specific to g2? I think it also applies to the ungaged sites, so I'm suggesting removing g2. Will need to update name elsewhere, too.
```suggestion
get_sb_data_log <- function(sb_var_ids, file_out) {
```

Reply:
Nope - good catch!